source/_integrations/recorder.markdown

```diff
@@ -371,6 +371,16 @@ sudo apt-get install default-libmysqlclient-dev libssl-dev
 pip3 install mysqlclient
 ```
 
+If Home Assistant is installed in a virtual environment, the additional libraries should be installed in the same virtual environment. If you installed Home Assistant Core to a Raspberry Pi following the [standard installation instructions](https://www.home-assistant.io/installation/raspberrypi) the following process should work.
```

Comment:
Let's rewrite this a little bit, the important point is to activate the venv before running pip3.
Also, the link should be relative.
```suggestion
If Home Assistant is installed in a virtual environment, the additional libraries should be installed in the same virtual environment. If you installed Home Assistant Core to a Raspberry Pi following the [standard installation instructions](/installation/raspberrypi) the venv can be activated like this prior to installing either `pymysql` or `mysqlclient`:
```